appeals/web/src/server/lib/mappers/data/types.d.ts

```diff
@@ -0,0 +1,16 @@
+import { MappedInstructions } from '../types';
+
+export type InitialiseAndMapDataFactory = <
```

Comment:
Similarly to my comment on `initialiseAndMapDataFactory`, the complexity of this type could be a worry in terms of understandability. It's true we're using TS already but there aren't many instances where it's this complex, and it's potentially a stumbling block for devs not as familiar with TS.

Reply:
Gunna have to refer to Biggs' comment 
> more complex for noobs, but hey ho.